client/src/partials/budget/analysis/analysis_budget.js

```diff
@@ -360,7 +360,9 @@ angular.module('bhima.controllers')
       });
 
       // Exportation
-      exportFile.csv(fileData, 'Budget Analysis Report');
+      var labelPeriod     = session.selectedPeriod || '';
+      var labelFiscalYear = session.selectedFiscalYear.start_year || '';
```

Comment:
Should you allow these to be undefined or false?  It is better to default to `''` than throw an error?